packages/modules/storybook-auth-addon/package.json

```diff
@@ -12,24 +12,24 @@
     "@bentley/bentleyjs-core": "2.11.0",
```

Comment:
really should be using the itwin auth client, not the old deprecated bentley 2.x pkgs

Reply:
Defer to a separate issue https://github.com/iTwin/admin-components-react/issues/109